test/integration/integration.ts

```diff
@@ -29,49 +19,4 @@ describe('integration tests', function () {
   it('isConnected', function () {
     assert(this.client.isConnected())
   })
```

Comment:
Since this is the only test left, and it's super tiny, is there a better place we could put it instead so we can just get rid of the general 'integration.ts' file? @mvadari

Reply:
Nevermind, I've returned the multisign test to integration.ts because the `submit_multisign` test is different (It won't work with the browser test because the browser doesn't use the rippled instance). For now integration.ts is basically just the browser test file.